NewForce-Cohort-9/tabloidmvc-stafford-infection-1
# Fw edit comment ticket22

# Description

Added functionality Edit Comment that allows you to edit a comment for a specific post. This allows you to edit a comment that you posted to correct mistakes or add additional content.


Trello Ticket: https://trello.com/c/VHHWRWsl/22-22-edit-a-comment


## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. Ensuring that you have cloned down the repo and added the database, git fetch --all in the master branch to retrieve my branch named fw-edit-comment-ticket22
2. git checkout fw-edit-comment-ticket22 to switch to my branch. If you don't see the most recent version of my files, git pull origin fw-edit-comment-ticket22
3. Open up Visual Studio. 
4. Run the application in a browser window
5. You should see the 'Home' page with a navigation bar at the top of the page. Select 'Login' and log in with this email: [admin@example.com]
6. Go to Posts on the nav bar
7. For testing purposes, look at the first post titled “C# is the Best Language”.
8. You will see an “eye” icon/button, click on it to go to Post Details 
9. Given you are viewing the Details of the Post, when you select the View Comments button (two way conversation messaging bubbles icon) 
10. You are directed to the Comments list page for the Post
11. Each Comment displays the following information: Subject, Content, Author's Display Name, Creation date, as well as edit and delete buttons if you are the author of the comment. 
12. You will find several comments by the admin user.
13. Click on the Edit option for the comment with Subject: “Test” and Content: “TestingPR”.
14. You are directed to another page where you will see input fields for Subject and Content and the existing content of these input fields.
15. You can make changes to the Subject and Content of the comment, such as adding “1” to the Content, Content: “TestingPR1”.
16. Click Save when done.
17. You will be directed back to the comment list.
18. Ensure that the comment that you edited has the new changes.

# Checklist:
